src/AutoRest.CSharp/Common/Output/Expressions/KnownValueExpressions/JsonSerializerExpression.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using AutoRest.CSharp.Common.Output.Expressions.ValueExpressions;
+using AutoRest.CSharp.Generation.Types;
+
+namespace AutoRest.CSharp.Common.Output.Expressions.KnownValueExpressions
+{
+    internal static class JsonSerializerExpression
```

Comment:
this is a static class because `JsonSerializer` is a static class, we do not need to introduce a record as expression for it.